tldw_Server_API/tests/AuthNZ/integration/test_single_user_claims_permissions.py

```
@@ -0,0 +1,266 @@
+from tldw_Server_API.app.core.AuthNZ.User_DB_Handling import User, get_request_user
+
+
+pytestmark = pytest.mark.integration
+
+
+router = APIRouter()
+
+
+@router.get("/single-user/whoami/user")
+async def whoami_user(
+    user: User = Depends(get_request_user),
```

Ruff (0.14.6)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)